src/core/accountAddress.ts

```diff
@@ -496,4 +496,17 @@ Received ${input}`,
     if (this.data.length !== other.data.length) return false;
     return this.data.every((value, index) => value === other.data[index]);
   }
+
+  /**
+   * Returns a lossless short string representation of the address by trimming leading zeros.
+   * If the address consists of all zeros, returns "0".
+   *
+   * @returns A string representation of the address without leading zeros
+   * @group Implementation
+   * @category Serialization
+   */
+  shortStrLossless(): string {
+    const hex = bytesToHex(this.data).replace(/^0+/, "");
+    return hex === "" ? "0" : hex;
+  }
```

Comment:
There's already a function for this?

`toString()`

Reply:
I don't think ToString will trim the leading 0s

Reply:
* Return the AccountAddress as a string as per AIP-40.
   * https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-40.md.
   * This representation returns special addresses in SHORT form (0xf)
   * and other addresses in LONG form (0x + 64 characters).